Premier league.ipynb

```diff
@@ -0,0 +1,1287 @@
+{
```

Comment:
<ol><li>Please increase resolutions by adding more <code>bins</code>  to the histogram.</li><li>This piece of code assumes the games are ordered by date. Is it True ? Please make the assumption explicit, or sort the games explicitly yourself.</li><li>The column "group" is important, but it has a mysterious name. Please use a more verbose name. You can find a name in Hebrew and then translate it.</li><li>Keep columns names conventions - columns are either upper case acronyms or CamelCase words.</li><li> Please use <code>alpha</code>  in the hist function to make multiple histograms visible simultaneously.</li><li>Which histogram belongs to which group ? Please use <code>label=f'{group}`</code>   and then add a <code>legend()</code>  (<code>pyplot.legend()</code>  should do it).</li><li>There are too many histograms on the same graph - limit the graph to a smaller number of groups</li></ol>
<br/>

 _Reply via <a href='https://app.reviewnb.com/A4Vision/games-course/pull/27/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

Reply:
<p>I not understand note 7. What does this mean?</p>

Reply:
<p>Simpler than you think. The plot s not readable because it contains too many graphs simultaneously.</p><p> </p><p>Therefore the graph is does not give visibility to any phenomena, even if a phenomena exists. This happened due to a mistake in the task definition.</p><p> </p>